### PR TITLE
ActiveSupport String inflections

### PR DESCRIPTION
**Pull request by hibariya**

### CODE REVIEW

**Comment by ksss** [comment]
```diff
$ bundle exec rbs diff --detail --format diff --type-name String --before gems/activesupport/6.0 --after 593
- [::String public] def safe_constantize: () -> untyped
+ [::String public] def safe_constantize: () -> untyped?

- [::String public] def titleize: (?keep_id_suffix: bool keep_id_suffix) -> untyped
+ [::String public] def titleize: (?keep_id_suffix: ::boolish) -> ::String

- [::String public] def demodulize: () -> untyped
+ [::String public] def demodulize: () -> ::String

- [::String public] def deconstantize: () -> untyped
+ [::String public] def deconstantize: () -> ::String

- [::String public] def parameterize: (?locale: untyped? locale, ?preserve_case: bool preserve_case, ?separator: ::String separator) -> untyped
+ [::String public] def parameterize: (?separator: ::String, ?preserve_case: ::boolish, ?locale: ::Symbol) -> ::String

- [::String public] def tableize: () -> untyped
+ [::String public] def tableize: () -> ::String

- [::String public] def classify: () -> untyped
+ [::String public] def classify: () -> ::String

- [::String public] def humanize: (?keep_id_suffix: bool keep_id_suffix, ?capitalize: bool capitalize) -> untyped
+ [::String public] def humanize: (?capitalize: ::boolish, ?keep_id_suffix: ::boolish) -> ::String

- [::String public] def upcase_first: () -> untyped
+ [::String public] def upcase_first: () -> ::String

- [::String public] def foreign_key: (?bool separate_class_name_and_id_with_underscore) -> untyped
+ [::String public] def foreign_key: (?::boolish separate_class_name_and_id_with_underscore) -> ::String
```

**Comment by ksss** [comment]
https://github.com/rails/rails/blob/v6.0.6.1/activesupport/lib/active_support/core_ext/string/inflections.rb
https://github.com/rails/rails/blob/v7.1.3.4/activesupport/lib/active_support/core_ext/string/inflections.rb

File: gems/activesupport/6.0/activesupport.rbs [hunk 1]
```diff
@@ -392,10 +392,23 @@ end
 
 class String
   def camelize: (?Symbol first_letter) -> String
+  alias camelcase camelize
   def underscore: () -> String
   def dasherize: () -> String
   def pluralize: (?Integer | Symbol? count, ?Symbol locale) -> String
   def singularize: (?Symbol locale) -> String
+  def constantize: () -> untyped
+  def safe_constantize: () -> untyped?
```
[the remark this answers is not in the document]
Comment (hibariya) [reply gems/activesupport/6.0/activesupport.rbs:401]: @ksss Thanks for taking a look Yuki san! Yeah, I thought about it as well 👍  but noticed that it actually could be anything. For example, `'RUBY_VERSION'.constantize` will be a string. Also, its value should basically the same as the value of [Module#const_get](https://github.com/ruby/rbs/blob/5833f015a670d5095e8a1576f398580f67ab1835/core/module.rbs#L569). That's why I left these return values alone.

File: gems/activesupport/6.0/activesupport.rbs [hunk 2]
```diff
@@ -392,10 +392,23 @@ end
 
 class String
   def camelize: (?Symbol first_letter) -> String
+  alias camelcase camelize
   def underscore: () -> String
   def dasherize: () -> String
   def pluralize: (?Integer | Symbol? count, ?Symbol locale) -> String
   def singularize: (?Symbol locale) -> String
+  def constantize: () -> untyped
```
[the remark this answers is not in the document]
Comment (hibariya) [reply gems/activesupport/6.0/activesupport.rbs:400]: @ksss Thanks for taking a look Yuki san! Yeah, I thought about it as well 👍  but noticed that it actually could be anything. For example, `'RUBY_VERSION'.constantize` will be a string. Also, its value should basically the same as the value of [Module#const_get](https://github.com/ruby/rbs/blob/5833f015a670d5095e8a1576f398580f67ab1835/core/module.rbs#L569). That's why I left these return values alone.

File: gems/activesupport/6.0/activesupport.rbs (hunk 1 above)
[the remark this answers is not in the document]
Comment (hibariya) [reply gems/activesupport/6.0/activesupport.rbs]: Updated: fdc3c445757bb0eef7c5f87d799bc0a510e00e36
@ksss Could you take a look again? Thanks!

File: gems/activesupport/6.0/activesupport.rbs (hunk 2 above)
[the remark this answers is not in the document]
Comment (ksss) [reply gems/activesupport/6.0/activesupport.rbs:400]: Make sense!

**Review by ksss — APPROVED** [review]
👌

**Pull request merged**